deploy/eck-stack/charts/eck-autoops/examples/basic.yaml

```diff
@@ -0,0 +1,24 @@
+---
+# Version of AutoOps Agent Policy.
+#
+version: 9.3.0-SNAPSHOT
+
+# AutoOps configuration values.
+# These values are required and will be used to create a Kubernetes Secret.
+#
+ccmApiKey: "your-api-key-here"
```

Comment:
secret values in a Helm values file seems like a big no-no to me.

Reply:
I've simply made the secretRef required in the Helm chart now.